### PR TITLE
Version to 1.2.0

### PR DESCRIPTION
**Pull request by SgtSilvio**
**Motivation**

**Changes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,23 +24,44 @@ backpressure support.
...
+  [MQTT 5.0](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html) features**
 - API flavors:
-  - Reactive, Async and Blocking
-  - Flexible switching
-  - Consistent and clearly separated
-- Backpressure support:
+  - **Reactive**: [Reactive Streams](https://www.reactive-streams.org/) compatible, 
+    [RxJava](https://github.com/ReactiveX/RxJava) and [Reactor](https://github.com/reactor/reactor-core)
+  - **Asynchronous**: futures and callbacks
+  - **Blocking**: quick start and testing
```
Comment (SgtSilvio) [inline README.md:33]:
```suggestion
  - **Blocking**: for quick start and testing
```

File: README.md [hunk 2]
```diff
@@ -24,23 +24,44 @@ backpressure support.
...
+  [MQTT 5.0](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html) features**
 - API flavors:
-  - Reactive, Async and Blocking
-  - Flexible switching
-  - Consistent and clearly separated
-- Backpressure support:
+  - **Reactive**: [Reactive Streams](https://www.reactive-streams.org/) compatible, 
+    [RxJava](https://github.com/ReactiveX/RxJava) and [Reactor](https://github.com/reactor/reactor-core) APIs available.
+  - **Asynchronous**: futures and callbacks
+  - **Blocking**: for quick start and testing
```
Comment (SgtSilvio) [inline README.md:33]:
```suggestion
  - **Blocking API**: for quick start and testing
```

File: README.md [hunk 3]
```diff
@@ -24,23 +24,44 @@ backpressure support.
...
-- All MQTT 3.1.1 and MQTT 5.0 features
+- **All MQTT [3.1.1](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/errata01/os/mqtt-v3.1.1-errata01-os-complete.html) and 
+  [MQTT 5.0](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html) features**
 - API flavors:
-  - Reactive, Async and Blocking
-  - Flexible switching
-  - Consistent and clearly separated
-- Backpressure support:
+  - **Reactive**: [Reactive Streams](https://www.reactive-streams.org/) compatible, 
+    [RxJava](https://github.com/ReactiveX/RxJava) and [Reactor](https://github.com/reactor/reactor-core) APIs available.
```
Comment (SgtSilvio) [inline README.md:31]:
```suggestion
    [RxJava](https://github.com/ReactiveX/RxJava) and [Reactor](https://github.com/reactor/reactor-core) APIs available
```

File: README.md [hunk 4]
```diff
@@ -24,23 +24,44 @@ backpressure support.
 
 ## Features
 
-- All MQTT 3.1.1 and MQTT 5.0 features
+- **All MQTT [3.1.1](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/errata01/os/mqtt-v3.1.1-errata01-os-complete.html) and
```
Comment (MicWalter) [inline README.md:27]: here you exclude MQTT from the link to the 3.1.1 spec but for MQTT 5 you include MQTT in the link

**Pull request merged**